docs/theming.md

```diff
@@ -185,9 +185,170 @@ extension MyViewController {
+extensions in https://github.com/material-components/material-components-ios/issues/7172.
+
+#### Typical migration
+
+When migrating from a subsystem Themer to a Theming extension the first thing to understand is that
+a Theming extension will apply *all* of the available subsystems to a given component. If you were
+previously relying on the ability to apply only one subsystem (e.g. Color) to a component, please
+file a
+[feature request](https://github.com/material-components/material-components-ios/issues/new/choose)
+with a code snippet of your existing use case.
+
+Generally-speaking, the migration from a subsystem Themer to a Theming extension will involve the
+following steps:
+
+1. Update your dependencies.
+2. Update your imports.
+3. Changes to your code.
+
+##### Update your dependencies
+
+In order to use theming extensions you'll need to first follow the
+[beta component instructions](../../contributing/beta_components.md).
+
+When a component has a theming extension it will always be available as a `Theming` target
+alongside the component. For example:
+
+In CocoaPods:
+
+```ruby
+// Old
+pod 'MaterialComponents/Cards'
+pod 'MaterialComponents/Cards+ColorThemer'
+
+// New
+pod 'MaterialComponents/Cards'
+pod 'MaterialComponents/Cards+Theming'
+```
+
+In Bazel:
+
+```ruby
+// Old
+  deps = [
+      "//components/schemes/Cards",
+      "//components/schemes/Cards:ColorThemer",
+  ],
+
+// New
+  deps = [
+      "//components/schemes/Cards",
+      "//components/schemes/Cards:Theming",
+  ],
+```
+
+##### Update your imports
+
+Replace any Themer import with the component's Theming import:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+// Old
+import MaterialComponents.MaterialCards_ColorThemer
+
+// New
+import MaterialComponents.MaterialCards_Theming
+```
+
+#### Objective-C
+
+```objc
+// Old
+#import <MaterialComponents/MaterialCards+ColorThemer.h>
+
+// New
+#import <MaterialComponents/MaterialCards+Theming.h>
+```
+<!--</div>-->
+
+##### Changes to your code
+
+Replace any Themer code with the equivalent use of a component's theming extension. Each Themer's
+equivalent Theming extension API is described in the Themer's API documentation.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+// Old
+let colorScheme = MDCSemanticColorScheme()
+MDCCardsColorThemer.applySemanticColorScheme(colorScheme, to: card)
+
+// New
+let scheme = MDCContainerScheme()
+card.applyTheme(withScheme: scheme)
+```
+
+#### Objective-C
+
+```objc
+// Old
+MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
+[MDCCardsColorThemer applySemanticColorScheme:colorScheme toCard:card];
+
+// New
+MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+[card applyThemeWithScheme:containerScheme];
+```
+<!--</div>-->
+
+If you made customizations to one of the subsystem schemes, you can now customize the container
+scheme's subsystem instances instead:
```

Comment:
Tip: this might be a good place to remind the reader that they only need to do that once if they share a single container scheme with multiple material components.